00-introduction.md

```diff
@@ -23,6 +23,62 @@ This is version 0.
 10. [BOLT #10](10-dns-bootstrap.md): DNS Bootstrap and Assisted Node Location
 11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
 
+## The Spark: A Short Introduction to Lightning
+
+Lightning is a protocol for fast payments using Bitcoin, using a
+network of channels.
+
+### Channels
+
+Lightning works by establishing *channels*: two participants share
+custody of some bitcoin (e.g. 0.1 bitcoin), spendable only with both
+their signatures.
+
+Initially they each hold a bitcoin transaction which sends all the
+bitcoin (e.g. 0.1) back to one party.  They can sign a new bitcoin
+transaction which splits these funds differently, e.g. 0.09 to one
+party, 0.01 to the other, and invalidate the previous bitcoin
+transaction so it won't be spent.
+
+See [BOLT #2: Channel Establishment](02-peer-protocol.md#channel-establishment) for more on
+channel establishment and [BOLT #3: Funding Transaction Output](03-transactions.md#funding-transaction-output) for the format of the bitcoin transaction which creates the channel.  See [BOLT #5: Recommendations for On-chain Transaction Handling](05-onchain.md) for the requirements when peers disagree or fail, and the cross-signed bitcoin transaction must be spent.
+
+### Conditional Payments
+
+The participants can add a conditional payment to the channel,
+e.g. "you get 0.01 bitcoin if you reveal the secret within 6 hours".
+Once the recipient presents the secret, that bitcoin transaction is
+replaced with one lacking the conditional payment and adding the funds
+to that recipient's output.
```

Comment:
This feels like it's missing a lot of context and dumps readers into the deep end without explaining why they're going there. Maybe if you lead with something like. "A Lightning channel only allows payment between two participants, but channels can be connected together to form a network that allows payments between all members of the network. This requires the technology of a conditional payment, which can be added to a channel, ..." [and then continue with line 49].

Reply:
Agreed.